Source/monster.cpp

```diff
@@ -2689,15 +2650,10 @@ void ButcherAi(int monsterId)
 		return;
 	}
 
-	int mx = monster.position.tile.x;
-	int my = monster.position.tile.y;
-	int x = mx - monster.enemyPosition.x;
-	int y = my - monster.enemyPosition.y;
-
-	Direction md = GetDirection({ mx, my }, monster.position.last);
+	Direction md = GetDirection({ monster.position.tile.x, monster.position.tile.y }, monster.position.last);

```

Comment:
It would be good if you could correct comment in general so I won't have to comment on every instance :)
```suggestion
	Direction md = GetDirection(monster.position.tile, monster.position.last);
```